src/installer/installer.go

```diff
@@ -164,6 +164,20 @@ func (i *installer) startBootstrap() error {
 	}
 	i.log.Info("Done extracting ignition to filesystem")
 
+	// reload systemd configurations from filesystem and regenerate dependency trees
+	i.log.Infof("Restarting NetworkManager.service")
```

Comment:
move this logic to function restartNetworkManager.
will be easier to rebase and unitest it